.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,11 @@
-Description:
-<!--
-Replace this template with your PR description.
-Please remember to keep in mind the security levels outlined in
-[CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md) and to
-include a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title
-
-* **no**: None
-* **low**: Low chance of potential impact to, or exposure of patient data
-* **moderate**: Moderate chance of potential impact to, or exposure of patient data
-* **severe**: Severe chance of potential impact to, or exposure of patient data
-
-Please also:
-
-* Get thumbs from reviewer(s)
-* Verify all tests go green, including CI tests
--->
-
+**Description**
 
 ---
 **PR checklist**
 
-- [ ] This PR meets the Acceptance Criteria in the JIRA story
-- [ ] The JIRA story has been moved to Dev Review
-- [ ] This PR includes appropriate unit tests
-- [ ] I have added explanatory comments where the logic is not obvious
-- [ ] I have run and tested this change locally, and my testing process is described here
+- [ ] If this PR is associated with a Jira issue, I have included an issue number tag in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md)
```

Comment:
This is new.

Reply:
```suggestion
- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md)
```
Suggesting "no ticket" may help us be consistent in case we ever want to search for these.

Reply:
Good idea, done